fdbserver/worker.actor.cpp

```diff
@@ -761,10 +766,10 @@ ACTOR Future<Void> monitorTraceLogIssues(Optional<Reference<AsyncVar<std::set<St
 			}
 		}
 		if (issues.present()) {
-			std::set<StringRef> _issues = getTraceLogIssues();
+			std::set<std::string> _issues = getTraceLogIssues();
 			if (pingTimeout) {
 				// Ping trace log writer thread timeout.
-				_issues.insert(LiteralStringRef("trace_log_writer_thread_likely_died"));
+				_issues.insert("trace_log_writer_thread_likely_died");
```

Comment:
Since this could be from a hung thread as well, it might make sense to call this `trace_log_writer_thread_unresponsive`. Ultimately we'd probably want to also report errors specific to the failure of the thread, if we can get them, in which case this message would be redundant and may need to be filtered in the case of a dead thread. For now, though, it's probably ok to have it cover both scenarios.